packages/near-membrane-base/src/blue.ts

```diff
@@ -57,10 +57,12 @@ function createBlueShadowTarget(target: BlueProxyTarget): BlueShadowTarget {
     if (typeof target === 'function') {
         // this new shadow target function is never invoked just needed to anchor the realm
         try {
+            // istanbul ignore next
             // eslint-disable-next-line func-names
             shadowTarget = 'prototype' in target ? function () {} : () => {};
```

Comment:
☝️ What's the reason to skip this one?

Reply:
I didn't add any additional comments here because it was already written on line 58 :D